cmd/geth/apicmd.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/rand"
+	"strconv"
+	"strings"
+
+	"gopkg.in/urfave/cli.v1"
+
+	"github.com/ethereumproject/go-ethereum/node"
+	"github.com/ethereumproject/go-ethereum/rpc"
+)
+
+var (
+	apiCommand = cli.Command{
+		Action:          execAPI,
+		Name:            "api",
+		Usage:           "Run any API command",
+		SkipFlagParsing: true,
```

Comment:
Just one final issue I found; skipping flag parsing disabled `--help` usage -- this can be fixed by just removing the line.

```shell
~/gocode/src/github.com/ethereumproject/go-ethereum tzdybal-api_command= ⟠ geth api --help
dial unix /Users/ia/Library/EthereumClassic/mainnet/geth.ipc: connect: no such file or directory

# for comparison
~/gocode/src/github.com/ethereumproject/go-ethereum tzdybal-api_command= ⟠ geth monitor --help
monitor [command options] [arguments...]

The Geth monitor is a tool to collect and visualize various internal metrics
gathered by the node, supporting different chart types as well as the capacity
to display multiple metrics simultaneously.


OPTIONS:
	--attach value	API endpoint to attach to (default: "ipc:/Users/ia/Library/EthereumClassic/mainnet/geth.ipc")
	--rows value	Maximum rows in the chart grid (default: 5)
	--refresh value	Refresh interval in seconds (default: 3)
```